amethyst_network/src/resources/connection.rs

```diff
@@ -52,19 +51,19 @@ impl NetConnectionPool{
         }
     }
 
-    pub fn connection_from_uuid(&self,uuid:&Uuid)->Option<&NetConnection>{
-        for c in self.connections{
+    pub fn connection_from_uuid(&self,uuid:&Uuid)->Option<NetConnection>{
+        for c in &self.connections{
```

Comment:
Lookup in `HashMap` with indices to sockets

Reply:
There's two keys, so I'm keeping the vec.